lib/jsonapi-serializers/serializer.rb

```diff
@@ -364,11 +364,20 @@ def self.find_recursive_relationships(root_object, root_inclusion_tree, results)
           is_collection = true
           object = serializer.has_many_relationships[unformatted_attr_name]
         end
+
         if !is_valid_attr
           raise JSONAPI::Serializer::InvalidIncludeError.new(
             "'#{attribute_name}' is not a valid include.")
         end
 
+        if attribute_name.include?("_")
```

Comment:
Sorry one last nitpick: please use single quotes here for consistency. Also, same in the spec `it` clause.